include/bluetooth/audio/audio.h

```diff
@@ -1,9 +1,1699 @@
+ *
+ *  @param stream Stream object
+ *  @param meta_count Number of metadata entries
+ *  @param meta Metadata entries
+ *
+ *  @return 0 in case of success or negative value in case of error.
+ */
+int bt_audio_stream_metadata(struct bt_audio_stream *stream,
+			     uint8_t meta_count, struct bt_codec_data *meta);
+
+/** @brief Disable Audio Stream
+ *
+ *  This procedure is used by a client to disable a stream.
+ *
+ *  This shall only be called for unicast streams, as broadcast streams will
+ *  always be enabled once created.
+ *
+ *  @param stream Stream object
+ *
+ *  @return 0 in case of success or negative value in case of error.
+ */
+int bt_audio_stream_disable(struct bt_audio_stream *stream);
+
+/** @brief Start Audio Stream
+ *
+ *  This procedure is used by a client to make a stream start streaming.
+ *
+ *  This shall only be called for unicast streams.
+ *  Broadcast sinks will always be started once synchronized, and broadcast
+ *  source streams shall be started with bt_audio_broadcast_source_start().
+ *
+ *  @param stream Stream object
+ *
+ *  @return 0 in case of success or negative value in case of error.
+ */
+int bt_audio_stream_start(struct bt_audio_stream *stream);
+
+/** @brief Stop Audio Stream
+ *
+ *  This procedure is used by a client to make a stream stop streaming.
+ *
+ *  This shall only be called for unicast streams.
+ *  Broadcast sinks cannot be stopped.
+ *  Broadcast sources shall be stopped with bt_audio_broadcast_source_stop().
+ *
+ *  @param stream Stream object
+ *
+ *  @return 0 in case of success or negative value in case of error.
+ */
+int bt_audio_stream_stop(struct bt_audio_stream *stream);
+
+/** @brief Release Audio Stream
+ *
+ *  This procedure is used by a client to release a unicast or broadcast
+ *  source stream.
+ *
+ *  Broadcast sink streams cannot be released, but can be deleted by
+ *  bt_audio_broadcast_sink_delete().
+ *  Broadcast source streams cannot be released, but can be deleted by
+ *  bt_audio_broadcast_source_delete().
+ *
+ *  @param stream Stream object
+ *  @param cache True to cache the codec configuration or false to forget it
+ *
+ *  @return 0 in case of success or negative value in case of error.
+ */
+int bt_audio_stream_release(struct bt_audio_stream *stream, bool cache);
+
+/** @brief Send data to Audio stream
+ *
+ *  Send data from buffer to the stream.
+ *
+ *  @note Data will not be sent to linked streams since linking is only
+ *  consider for procedures affecting the state machine.
+ *
+ *  @param stream Stream object.
+ *  @param buf Buffer containing data to be sent.
+ *
+ *  @return Bytes sent in case of success or negative value in case of error.
+ */
+int bt_audio_stream_send(struct bt_audio_stream *stream, struct net_buf *buf);
+
+/** @brief Create audio unicast group.
+ *
+ *  Create a new audio unicast group with one or more audio streams as a
+ *  central. Streams in a unicast group shall share the same interval, framing
```

Comment:
Previously, "client" and "server" has been used.  Should this "central" be "server"?
(Ditto in next function.)

Reply:
Changed to `client`. Should it be more specific and be `unicast client`?

Reply:
"client" is understandable - the unicast client is the only client we have (here), so it must be that one.
"unicast client" is even better though, as that is the term elsewhere, so I would prefer that.